system/lib/libc/musl/arch/emscripten/bits/alltypes.h

```diff
@@ -86,9 +86,9 @@ typedef long suseconds_t;
 #ifdef __EMSCRIPTEN__
 // For canvas transfer implementation in Emscripten, use an extra 10th control field
```

Comment:
Addressed in https://github.com/kripken/emscripten/pull/4813/commits/d08eb3da9f8f7fb7bea02fba1ac97e455eb63cc3
... And moved `_b_lock` and `_b_waiters` back to where they were before that commit in https://github.com/kripken/emscripten/pull/4813/commits/c360ca5d3b0a7578adb6c8a0b45a1cd2ae3d3f16